arviz/stats/diagnostics.py

```diff
@@ -530,7 +543,11 @@ def ks_summary(pareto_tail_indices):
     df_k : dataframe
       Dataframe containing k diagnostic values.
     """
-    kcounts, _ = np.histogram(pareto_tail_indices, bins=[-np.Inf, 0.5, 0.7, 1, np.Inf])
+    _numba_flag = Numba.numba_flag
```

Comment:
Can this be set at the top once instead of being called in every func?

Reply:
Would that work?

Reply:
It's only being called twice; will setting this it at the top make any difference?

Reply:
If its only being called twice then I don't think its worth it